hlx_statics/scripts.js

```diff
@@ -1181,129 +1297,95 @@ let $CURRENT_API_FILTERS = [];
-                  $dropdownPopover.classList.remove('is-open');
-                  $dropdownPopover.ariaHidden = false;
-                }
-              });
-            } else if($button.id.indexOf('nav-profile-dropdown-button') >=0 ) {
-
-              let $profileDropdownPopover = $currentHeader.querySelector('div#nav-profile-dropdown-popover');
-
-              $button.addEventListener('click', (evt) => {
-                if(!evt.currentTarget.classList.contains('is-open')){
-                  $button.classList.add('is-open');
-                  $profileDropdownPopover.classList.add('is-open');
-                  $profileDropdownPopover.ariaHidden = false;
-                } else {
-                  $button.classList.remove('is-open');
-                  $profileDropdownPopover.classList.remove('is-open');
-                  $profileDropdownPopover.ariaHidden = false;
-                }
-              });
-            }
-          });
 
-          let $signIn = $header.querySelector('#signIn');
-          $signIn.addEventListener('click', (evt) => {
-            adobeIMSMethods.signIn();
-          });
-          
-          setActiveTab();
-          window.adobeIMSMethods.getProfile();
-          focusRing($header);
-        });
-      }
-    });
-  }
+        setActiveTab();
+        window.adobeIMSMethods.getProfile();
+        focusRing($header);
 
-  function decorateAnnouncement() {
-    decorateButtons('.announcement-container');
-    document.querySelectorAll(".announcement").forEach(($announcement) => {
-      removeEmptyPTags($announcement);
+        const $searchDropdownPopover = $header.querySelector('div.nav-console-search-frame');
+        const searchFrameOnLoad = () => {
+          $header.querySelectorAll('#nav-dropdown-search').forEach(($button) => {
+            $button.style.visibility = "visible";
 
-      $announcement.querySelectorAll('h1, h2, h3, h4, h5, h6').forEach(($h) => {
-        $h.classList.add('spectrum-Heading', 'spectrum-Heading--sizeM', 'announce-heading');
+            $button.addEventListener('click', (evt) => {
+              if (!evt.currentTarget.classList.contains('is-open')) {
+                $button.classList.add('is-open');
+                $searchDropdownPopover.style.visibility = 'visible';
+                document.body.style.overflow = 'hidden';
+              } else {
+                $button.classList.remove('is-open');
+                $searchDropdownPopover.style.visibility = 'hidden';
+                document.body.style.overflow = 'auto';
+              }
+            });
+          })
+        }
+        const queryParams = getQueryStringParameter('query');
+        const queryString = getQueryString();
+        const searchFrame = document.createElement('iframe');
+        searchFrame.id = "nav-search-iframe";
+        searchFrame.onLoad = searchFrameOnLoad();
+        searchFrame.src = `https://developer-stage.adobe.com/search-frame/${queryParams ? `?${queryString}` : ''}`;
```

Comment:
Need to add a way to know if it's on stage or prod.

Reply:
Added context switching in commit below.